packages/vue/src/TolgeeMixin.ts

```diff
@@ -1,38 +1,38 @@
 import { defineComponent } from 'vue';
 import { Subscription } from '@tolgee/core/lib/services/Subscription';
-import {
-  TolgeeContext,
-  TranslateFnProps,
-  TranslationParameters,
-} from './types';
+import { TolgeeContext, TranslateFnProps } from './types';
+import { TranslationParams } from '@tolgee/core';
 
 export const TolgeeMixin = defineComponent({
   inject: ['tolgeeContext'],
   beforeCreate() {
     this.$options._tolgee = {
       subscribed_keys: [] as string[],
```

Comment:
This snake case is some kind of Vue convention?

Reply:
No, but it felt better than `_subscribedKeys`. But on second look it would be probably better

Reply:
I'll change it. Don't know why I did it this way :D